minicluster/src/main/java/alluxio/master/LocalAlluxioCluster.java

```diff
@@ -129,6 +130,7 @@ public void initConfiguration() throws IOException {
       ServerConfiguration.set(entry.getKey(), entry.getValue());
     }
     ServerConfiguration.set(PropertyKey.TEST_MODE, true);
+    ServerConfiguration.set(PropertyKey.MASTER_METASTORE, MetastoreType.HEAP);
```

Comment:
can we add a comment for why we use this as the default for tests but not production?

Reply:
Updated, thanks.